Tests/GRPCProtobufCodeGenTests/ProtobufCodeGenParserTests.swift

```diff
@@ -105,89 +105,89 @@ final class ProtobufCodeGenParserTests: XCTestCase {
       CodeGenerationRequest.Dependency(module: "GRPCProtobuf")
     )
   }
+}
 
-  var helloWorldFileDescriptor: FileDescriptor {
-    let requestType = Google_Protobuf_DescriptorProto.with {
-      $0.name = "HelloRequest"
-      $0.field = [
-        Google_Protobuf_FieldDescriptorProto.with {
-          $0.name = "name"
-          $0.number = 1
-          $0.label = .optional
-          $0.type = .string
-          $0.jsonName = "name"
-        }
-      ]
-    }
-    let responseType = Google_Protobuf_DescriptorProto.with {
-      $0.name = "HelloReply"
-      $0.field = [
-        Google_Protobuf_FieldDescriptorProto.with {
-          $0.name = "message"
-          $0.number = 1
-          $0.label = .optional
-          $0.type = .string
-          $0.jsonName = "message"
-        }
-      ]
-    }
+var helloWorldFileDescriptor: FileDescriptor {
```

Comment:
Generally avoid putting properties into the global namespace, it's rarely the right way organise code. 

In this case it'd be better to have an extension on `Google_Protobuf_DescriptorProto`:

```swift
extension Google_Protobuf_DescriptorProto {
  static var helloWorld: Google_Protobuf_DescriptorProto {
    // ...
  }
}
```